src/taipy/core/data/_data_model.py

```diff
@@ -52,6 +53,7 @@ def from_dict(data: Dict[str, Any]):
             parent_ids=data["parent_ids"],
             last_edit_date=data.get("last_edit_date", data.get("last_edition_date")),
             job_ids=data["job_ids"],
+            edits=data["edits"],
```

Comment:
What about something like the following?
```suggestion
            edits=data.get("edits", __to_edit_list(data.get("job_ids")))
```
So basically, when reading an existing data model : 
- if `edits` does not exist, we build it on the fly from `job_ids`. And next time it is written, it will not contain `job_ids` anymore.
- if `edits` already exist, that means the last time the data node was written was with the new version, so job_ids does not exist anymore.

Reply:
That's correct and that is what the code will be doing, but the PR is currently still in progress and the code is not pushed yet.
Notice that this PR is not ready for review yet and is still in "draft" ;-)